docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst

```diff
@@ -180,13 +159,11 @@ Ansible's "Network Fact modules" gather information from the system and store th
 To ensure we call the correct mode (``*_facts``) the task is conditionally run based on the group defined in the inventory file, for more information on the use of conditionals in Ansible Playbooks see :ref:`the_when_statement`.
 
 
-Example
-=======
+Step 1: Creating the inventory
+------------------------------
 
 In this example, we will create an inventory file containing some network switches, then run a playbook to connect to the network devices and return some information about them.
```

Comment:
I might move this sentence up above the `Step 1` title, since it's an overview of the entire example.